sdk/synapse/synapse/src/synapseClientContext.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+import * as Models from "./models";
+import * as msRest from "@azure/ms-rest-js";
+import * as msRestAzure from "@azure/ms-rest-azure-js";
+
+const packageName = "@azure/synapse";
+const packageVersion = "1.0.0";
```

Comment:
Now that the package.json is updated, this value should be updated as well (I think if the package were regenerated both values would match).
```suggestion
const packageVersion = "1.0.0-preview.1";
```